src/obr/signac_wrapper/operations.py

```diff
@@ -260,9 +260,48 @@ def dispatch_pre_hooks(operation_name, job):
     execute_pre_build(operation_name, job)
 
 
+def gather_caseFiles(case_path: str) -> list[str]:
+    """Gathers a list of files inside a case folder that are also not symlinks."""
+    sys_path = case_path + '/system'
```

Comment:
I think we could simplify this, by using the OpenFOAMCase class in https://github.com/hpsim/OBR/blob/dev/src/obr/OpenFOAM/case.py.

Maybe we should add a property to the OpenFOAMCase like
```
@property
def config_file_tree(self) -> list[str}:
```

which return a list of existing files?